iOS/DuckDuckGo/DefaultOmniBarViewController.swift

```diff
@@ -58,6 +60,7 @@ final class DefaultOmniBarViewController: OmniBarViewController {
 
     override func textFieldShouldBeginEditing(_ textField: UITextField) -> Bool {
         if aiChatSettings.isAIChatSearchInputUserSettingsEnabled {
+            omniDelegate?.onExperimentalAddressBarTapped()
```

Comment:
This will be fired on every first responder attempt (even programmatic) so you need to check `OmniBarViewController.textFieldTapped` property. See `MainViewController.onTextFieldWillBeginEditing`.